app/templates/drupal/openatrium/project-main.make.yml

```diff
@@ -17,9 +17,8 @@ includes:
   ## Project-specific Dependencies
   - <%= projectName %>.make.yml
 
-## Open Atrium
-
 projects:
+  # Base Distribution - [Open Atrium](https://www.drupal.org/project/openatrium)
   openatrium:
     type: profile
     version: "<%= drupalDistroRelease %>"
```

Comment:
Some of the atrium changes were made as part of aligning the existing Atrium "plugin" to the way the Lightning "plugin" came together. In a couple cases, it was a bug discovered in code shared by Lightning and Atrium.

We have sufficiently few contributors that unless you think the git history could be confusing, we can probably move forward as-is unless one of the changes is a blocker.